.github/workflows/csv_scrape.yml

```diff
@@ -0,0 +1,61 @@
+name: CSV scrape automation hourly
+
+on:
+    schedule:
+        - cron: '0 * * * *'
+    push:
+      branches: [ "main" ]
+    pull_request:
+        branches: [ "main" ]
```

Comment:
I assume you've tested this on the pr branch.

Reply:
Yeah so I have it on the cron schedule and I also have this running on pull requests and I put it on workflow dispatch (meaning it can manually be activated via GitHub) I think it's good for it to be running as much as possible since sometimes the data isn't updated on the hour exactly so this just gives us more updates to the database